site2/website/release-notes.md

```diff
@@ -1,11 +1,98 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.7.2 &mdash; 2021-04-XX <a id=“2.7.2”></a>
+
+#### Broker
+- Fix the useless retry when the maximum number of subscriptions is reached [#9991](https://github.com/apache/pulsar/pull/9991)
+- wrong timeunit in updating lastLedgerCreationInitiationTimestamp [#10049](https://github.com/apache/pulsar/pull/10049)
+- Avoid spammy logs in case of BK problems [#10088](https://github.com/apache/pulsar/pull/10088)
+- Fix NonDurableCursorImpl initialPosition by startCursorPosition greater than lastConfirmedEntry problem. [#10095](https://github.com/apache/pulsar/pull/10095)
+- fix 8115 Some partitions get stuck after adding additional consumers to the KEY_SHARED subscriptions [#10096](https://github.com/apache/pulsar/pull/10096)
```

Comment:
Shall we just abstract the PR title, or refine it accordingly in release notes?
If we just abstract PR titles, it's ok to write it here. Otherwise, we'd better refine it.

Reply:
Same comment for others.

Reply:
I extracted the PR titles with a script and I fixed the most visible errors or added a little context in order to allow the reader to understand a little what's the PR is about.

Reply:
@Jennifer88huang what can we do ? Refine more accurately all of the titles ?

(please note that I closed this PR in favour of #10550 due to a problem with GitHub)